pages/api/notification.ts

```diff
@@ -26,31 +37,42 @@ const transporter = nodemailer.createTransport({
 
 const emails = new Map<
   keyof Events,
-  ((data: Events[keyof Events]) => SendMailOptions | null)[]
+  ((data: any) => SendMailOptions | null | Promise<SendMailOptions | null>)[]
```

Comment:
I don't think any email function need to be async anymore:
```suggestion
  ((data: any) => SendMailOptions | null>)[]
```

Reply:
indeed this was required because the data were not in the webhook but this is now solved so I simplified the logic [c4f5360](https://github.com/liteflow-labs/starter-kit/pull/276/commits/c4f53604010f2758c2ba25faaa4fc5da6e4878b3)